### ISSUE TITLE
Ellaboration of a descriptive manual of the purpose of the fields in each tab

### ISSUE THREAD

**Issue by febrezo** [cyb3rfox/Aurora-Incident-Response#63]
Additional docs to help newcomers on using the application might be required.

**Comment by cyb3rfox** [comment]
Yes the documentation side is bad, but there will be more people who know how to use it. We will strongly use the tool in the upcoming SANS FOR608 class.

**Comment by de3c0d3r** [comment]
I'm new here, but I'd be happy to test any beta or on going project and provide documentation
